collections/ansible_collections/cloudkit/templates/roles/ocp_4_17_small/tasks/main.yaml

```diff
@@ -1,5 +1,5 @@
 - name: Create a new instance of this template
-  when: cluster_template_state == 'present'
+  when: cluster_template_state == 'present' and cluster_template_stage == 'install'
```

Comment:
We're not setting `cluster_template_stage` anywhere, so I think this conditional will mean we never run the installation tasks.